package.json

```diff
@@ -722,6 +722,21 @@
           "markdownDescription": "`#editor.lineNumbers#` is determined by the active Vim mode, absolute when in insert mode, relative otherwise.",
           "default": false
         },
+        "vim.smartQuotes.enable": {
+          "type": "boolean",
+          "markdownDescription": "Enable smart quotes movements (similar to targets.vim)",
+          "default": true
+        },
+        "vim.smartQuotes.breakThroughLines": {
+          "type": "boolean",
+          "markdownDescription": "Whether to break through lines when using [n]ext/[l]ast motion",
```

Comment:
`markdownDescription` should always end with a period

Reply:
```suggestion
          "markdownDescription": "Whether to break through lines when using [n]ext/[l]ast motion.",
```